package.json

```diff
@@ -23,13 +23,13 @@
   ],
   "scripts": {
     "setup": "npm run build",
-    "start": "npm run setup && run-p server esbuild:watch",
-    "esbuild:watch": "pulp -w --then 'npm run esbuild' build -m Insect",
+    "start": "node copy && run-p server esbuild:watch",
+    "esbuild:watch": "spago build -w --then 'npm run esbuild'",
     "esbuild": "esbuild output/Insect/index.js --bundle --minify --outfile=web/insect.js --target=es6 --global-name=Insect",
```

Comment:
I could've used `spago bundle-app` here, but I like that we can pass whatever arguments to `esbuild` we want (I think we cannot do that with `spago bundle-app`).